data/json/furniture_and_terrain/furniture-appliances.json

```
@@ -876,19 +877,94 @@
         { "item": "xl_wind_turbine", "count": 1 }
       ]
     },
+    "bash": {
+      "str_min": 16,
+      "str_max": 40,
+      "sound": "whack!",
+      "sound_fail": "clang!",
+      "items": [
+        { "item": "solder_wire", "charges": [ 10, 20 ] },
+        { "item": "2x4", "count": [ 1, 30 ] },
+        { "item": "scrap", "count": [ 3, 6 ] },
+        { "item": "cable", "charges": [ 10, 15 ] },
+        { "item": "scrap", "count": [ 4, 6 ] },
+        { "item": "pipe", "count": [ 0, 3 ] }
+      ],
+      "//": "destroy_threshold equal to str_min instead of str_max due to delicate electronics",
+      "ranged": { "reduction": [ 8, 16 ], "destroy_threshold": 16, "block_unaimed_chance": "25%" }
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_water_wheel_unit",
+    "looks_like": "f_water_mill",
```

```suggestion
    "looks_like": "f_water_mill_active",
```
It should move in dead people